### ISSUE TITLE
Can not set <space> as leader key

### ISSUE THREAD

**Issue by amahmod** [onivim/oni2#2935]
```"vim.leader": "<space>"``` does not work.  But it works when i set the leader key to comma(,)  or any other key.

OS: Arch Linux (64 bit)
WM: BSPWM

My config

```json
{
  "workbench.colorTheme": "Solarized Dark",
  "workbench.activityBar.visible": true,
  "workbench.editor.showTabs": true,
  "workbench.iconTheme": "vs-seti",
  "workbench.sideBar.location": "left",
  "workbench.sideBar.visible": true,
  "workbench.statusBar.visible": true,
  "workbench.tree.indent": 2,
  "editor.detectIndentation": true,
  "editor.fontFamily": "Bitstream Vera Sans Mono",
  "editor.fontSize": 16,
  "editor.fontSmoothing": "antialiased",
  "editor.largeFileOptimizations": true,
  "editor.highlightActiveIndentGuide": true,
  "editor.lineNumbers": "relative",
  "editor.indentSize": 2,
  "editor.insertSpaces": true,
  "editor.matchBrackets": true,
  "editor.minimap.enabled": true,
  "editor.minimap.maxColumn": 120,
  "editor.minimap.showSlider": true,
  "editor.renderIndentGuides": true,
  "editor.renderWhitespace": "none",
  "ui.zoom": 1.1,
  "editor.rulers": [],
  "editor.tabSize": 2,
  "editor.zenMode.hideTabs": true,
  "editor.zenMode.singleFile": true,
  "files.exclude": [ "_esy", "node_modules", ".git" ],
  "vim.leader": "<space>",
  "vim.useSystemClipboard": [ "yank" ],
  "vim.highlightedyank.enable": true,
  "vim.highlightedyank.color": "#FF00FFFF",
  "go.useLanguageServer": true,
  "git.suggestSmartCommit": false
}
```

**Comment by bryphe** [comment]
Thanks for the issue @amahmod - I can reproduce 

Looks like a regression in the way we are handling the space key - working on a fix in #2941

**Comment by bryphe** [comment]
Fixed with #2941
